simulation/src/test/java/com/palantir/dialogue/core/SimulationTest.java

```diff
@@ -453,9 +468,27 @@ public static void afterClass() throws IOException {
                             throw new RuntimeException(e);
                         }
                     })
-                    .sorted(Comparator.comparing(String::trim))
+                    .collect(Collectors.joining("", "```\n", "```\n"));
+
+            String rows = files.stream()
+                    .filter(p -> p.toString().endsWith("png"))
+                    .map(p -> {
+                        String githubUrl = "https://raw.githubusercontent.com/palantir/dialogue/"
```

Comment:
😮

Reply:
I know, it's kinda heartbreaking. If you scroll through the diff of https://github.com/palantir/dialogue/pull/350/files, a bunch of pngs have changed, but github doesn't render them for me :( :(